.gitignore

```diff
@@ -1,5 +1,7 @@
-config.php
 .htaccess
 .DS_Store
 *.bak
 *.txt
+/config.php
+/scratch.php
```

Comment:
With a slash "/" in front, doesn't this reference files in the root of the drive, i.e. actual "/", where no files ever really are, just directories like  usr, bin, var, etc?

Reply:
Yep. My and/or PhpStorm's "mistake". 

Okay, let's call this sloppiness on my part. Will edit .gitignore to remove these two entries.

Oh good, .gitignore can have comments. :-)  I will comment that config.php should be ignored as one does not want to be uploading a fully configured, with passwords, etc. to GitHub. I know this first hand. How do I know first hand? I loaded an in-development config.php up to Github. No issues. And then 15 minutes later, Twilio emailed me saying that it had found my credentials in a repo and therefore had changed my authentication code.

The scratch.php was a scratch file trying out snippets. Both acceptable for a .gitignore file but should not be set up to ignore in the root directory.

When I told PhpStorm to ignore these two files, it somehow thought I was referring to files in the root.

Reply:
>  I will comment that config.php should be ignored as one does not want to be uploading a fully configured, with passwords, etc. to GitHub.

Yes.  Though `config.php` was already in the .gitignore file so that's odd.